wrappers/jira_client.py

```diff
@@ -75,21 +82,17 @@ def check_issue(self, issue):
         return result
 
     def get_worklogs(self, date_window):
-        params = {
-            'jira_url': self.jira_url,
-            'tempo_api': self.tempo_api,
-            'date_from': date_window.start.date().isoformat(),
-            'date_to': date_window.stop.date().isoformat(),
-            'tempo_user': self.tempo_user,
-        }
-
-        url = "{jira_url}{tempo_api}worklogs" \
-              "?username={tempo_user}&dateFrom={date_from}&dateTo={date_to}"
-
-        response = self.session.get(url.format(**params))
+        response = self.session.get(
+            self.worklog_url,
+            params={
+                "worker": self.account_id,
+                "from": date_window.start.date().isoformat(),
+                "to": date_window.stop.date().isoformat()
+            }
```

Comment:
To update on API v4

https://apidocs.tempo.io/#tag/Worklogs/operation/getWorklogsByAccount

-> query worklogs/account/{account_key} 

params {"from": start, "to": end, offset:0, limit: 50} -> you need to handle pagination if the number of entries == limit.